singleprocess.go

```diff
@@ -161,6 +161,8 @@ func (e *singleprocessEnv) RegisterComponentToStart(_ context.Context, _ string,
 }
 
 func (e *singleprocessEnv) GetComponentsToStart(context.Context, *call.Version) ([]string, *call.Version, error) {
+	// blocks forever to avoid excessive CPU consumption in weavelet.watchComponentsToStart
```

Comment:
Thanks, this is even more concise.

Could you just change the comment to:

`// Block forever since there's not going to be anything to start.`